src/signals/incident-management/containers/MyFilters/components/FilterItem/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import get from 'lodash.get';
+
+import FilterTagList from '../../../FilterTagList';
+
+import './style.scss';
+
+const getId = (filter) => {
+  const href = get(filter, '_links.self.href') || '';
+  const found = href.match(/\/(\d+)$/);
+  return (found && found[1]) || 0;
+};
+
+const handleApplyFilter = (filter, onApplyFilter, onRequestIncidents, onClose) => {
+  onApplyFilter(filter);
+  onRequestIncidents({ filter });
```

Comment:
I think it'll be nicer if the request incidents fetch code gets the active filters itself. So a `yield getState()` (or whatever the syntax) and use the active filters from there. That will remove the concern of which filters are active from most of your other code.

Reply:
You may even have a saga that auto fetches new incidents when the filters change. That way you aren't even concerned with fetching incidents here.

Reply:
ja Ruurd ik heb de RequestIncidents in de apply filter saga gestopt: veel beter